pkg/varlinkapi/pods.go

```diff
@@ -0,0 +1,237 @@
+package varlinkapi
+
+import (
+	"encoding/json"
+	"syscall"
+
+	"github.com/projectatomic/libpod/cmd/podman/shared"
+	"github.com/projectatomic/libpod/cmd/podman/varlink"
+	"github.com/projectatomic/libpod/libpod"
+)
+
+// CreatePod ...
+func (i *LibpodAPI) CreatePod(call iopodman.VarlinkCall, name, cgroupParent string, labels map[string]string) error {
```

Comment:
This should probably accept a struct like container create does - I can see us adding a lot more arguments in the future

Reply:
fixed